src/components/NodeTotalChart/CustomTick.tsx

```diff
@@ -0,0 +1,30 @@
+import { styled } from '@mui/material';
+
+const StyledLabel = styled('tspan')({
+  fontFamily: "Roboto",
```

Comment:
Correct me if I'm wrong, but I don't think we have Roboto font imported from anywhere.  I don't see it via CDN, dep, or local. I added a `cursive` font fallback and it used the fallback font.

Reply:
Good catch. Fixed in 5094d72